conjure-core/src/test/java/com/palantir/conjure/defs/validator/FieldNameValidatorTest.java

```diff
@@ -56,17 +56,19 @@ public void testInvalidNames() throws Exception {
             assertThatThrownBy(() -> FieldNameValidator.validate(FieldName.of(invalid)))
                     .isInstanceOf(IllegalArgumentException.class)
                     .hasMessageContaining(
-                            String.format("FieldName \"%s\" must follow one of the following patterns", invalid));
+                            String.format("FieldName \"%s\" must follow one of the following patterns: ", invalid))
+                    .hasMessageContaining("LOWER_CAMEL_CASE")
+                    .hasMessageContaining("[A-Z]");
```

Comment:
Let's use a single  `hasMessageContainingAll` so that on failure, we get the full context:
```java
hasMessageContainingAll(
    String.format("FieldName \"%s\" must follow one of the following patterns: ", invalid),
    "LOWER_CAMEL_CASE",
    "[A-Z]");